server/WebAPI/Controllers/AuthController.cs

```diff
@@ -32,5 +32,21 @@ public IActionResult Auth(AuthRequest authRequest)
             var token = _authService.MakeToken(user);
             return Ok(new AuthResponse { AccessToken = token });
         }
+
+        [AllowAnonymous]
+        [HttpPost]
+        [Route("SignUp")]
+        public IActionResult Registration(AuthRequest authRequest)
+        {
+            if (_userService.FindUser(authRequest.Email))
+            {
+                return UnprocessableEntity();
```

Comment:
please, can you tell more details about UnprocessableEntity?

Reply:
Because for example, I looked what error the github gives in the same situation, if the user exists, then - 422. And I used UnprocessableEntity()

Reply:
ok